### PR TITLE
Small improvements to k8s docs

### PR DESCRIPTION
**Pull request by andygrove**
I was able to deploy a cluster to EKS using these instructions but I couldn't figure out where the files were downloaded and couldn't get the example to run. I made some changes to change the path from `/mnt/nztaxi.csv` to `/mnt/nyctaxi/csv` for consistency with other published documentation, blog posts, demos, etc and will try again once this is merged to main.

### CODE REVIEW

**Comment by andygrove** [comment]
@jorgecarleitao fyi

File: kubernetes/README.md [hunk 1]
```diff
@@ -37,22 +37,28 @@ kubectl apply -f kubernetes/populate-data.yaml
...
+_Note that we also have publicly available Docker images hosted on [Dockerhub](https://hub.docker.com/u/ballistacompute) 
+for released versions of Ballista and these do not require authentication._
+
 
 ```bash
 kubectl create secret docker-registry github \
     --docker-server=docker.pkg.github.com \
     --docker-username=$GITHUB_USERNAME \
     --docker-password=$GITHUB_TOKEN \
-    --docker-email=jorgecarleitao@gmail.com
```
Comment (jorgecarleitao) [inline kubernetes/README.md:52]: AHAHA fuck, sorry about that!

File: kubernetes/README.md [hunk 2]
```diff
@@ -37,22 +37,28 @@ kubectl apply -f kubernetes/populate-data.yaml
...
 shows status `Completed`. We can't deploy the cluster before this step is completed, as the volume can only be claimed by
-a single pod at the time. Feel free to change [populate-data.yaml](./populate-data.yaml) do dump a smaller CSV.
+a single pod at the time. Feel free to change [populate-data.yaml](./populate-data.yaml) to dump a smaller CSV.
 
 ### Setup credentials for pulling images
 
-We offer docker images on github that we can use to run the cluster.
-They require authentication to be pulled. For this, we will create a secret in k8s:
+Development builds of the Ballista docker images are hosted on Github. It is necessary to set up authentication to be
+able to use these images. Use the following command to create a Kuberneted secret containing your github credentials.
```
Comment (jorgecarleitao) [inline kubernetes/README.md:45]: Kuberneted -> Kubernetes

**Comment by jorgecarleitao** [comment]
@andygrove went through this. Really good. Great that you were able to run these on EKS; I ran it on AKS and it also work, so that's good.

**Pull request merged**